src/algebra/parity.lean

```diff
@@ -9,6 +9,7 @@ import algebra.algebra.basic
 import algebra.group_power.basic
 import algebra.field_power
 import algebra.opposites
+import data.int.sqrt
```

Comment:
This import sounds like it's going the wrong way around. Can you instead move `nat.decidable_pred.is_square` and `int.decidable_pred.is_square` to `data.nat.parity`/`data.int.parity`?

Reply:
Sure!